packages/conf-mingw-w64-zlib-i686/conf-mingw-w64-zlib-i686.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+synopsis: "Virtual package for zlib on i686 mingw-w64 (32-bit x86)"
+description: "Ensures the i686 version of zlib for the mingw-w64 project is available"
+maintainer: "David Allsopp <david@tarides.com>"
+license: "GPL-1.0-or-later"
```

Comment:
The `zlib` library is distributed under the zlib license.

```suggestion
license: "zlib"
```

Unless the mingw64 version has a different license?

Reply:
My bad, fixed.